### PR TITLE
Use https instead of http in urls

### PR DESCRIPTION
**Pull request by alanbato**
This PR changes every instance of an http url to its https version for security reasons.

### CODE REVIEW

File: 3rdparty/jvm/com/google/guava/BUILD [hunk 1]
```diff
@@ -19,7 +19,7 @@ REV = '16.0'
 jar_library(name='guava',
   jars=[
     jar(org='com.google.guava', name='guava', rev=REV,
-        apidocs='http://docs.guava-libraries.googlecode.com/git-history/v%s/javadoc/' % REV),
+        apidocs='https://docs.guava-libraries.googlecode.com/git-history/v%s/javadoc/' % REV),
```
Comment (xjenniferxie) [inline 3rdparty/jvm/com/google/guava/BUILD]: Link is broken.

Comment (Eric-Arellano) [reply 3rdparty/jvm/com/google/guava/BUILD]: Indeed, this the warning I get when going to the site:

Screen Shot 2019-06-06 at 3 31 38 PM

This looks like the right link: https://javadoc.scijava.org/Guava/. I couldn't find the version control for the API, so you would remove the `% REV` piece.

**Review by Eric-Arellano — COMMENTED** [review]
Thanks!

File: build-support/ivy/ivy.xml [hunk 2]
```diff
@@ -18,8 +18,8 @@ limitations under the License.
 -->
 
 <ivy-module version="2.0"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
+            xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+            xsi:noNamespaceSchemaLocation="https://ant.apache.org/ivy/schemas/ivy.xsd">
```
Comment (stuhood) [inline build-support/ivy/ivy.xml]: This seems to break CI with:
```
 Exception in thread "main" java.text.ParseException: [xml parsing: ivy.xml:22:88: cvc-complex-type.3.2.2: Attribute 'xsi:noNamespaceSchemaLocation' is not allowed to appear in element 'ivy-module'. in file:/home/travis/build/twitter/commons/build-support/ivy/ivy.xml
 ]
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser$AbstractParser.checkErrors(AbstractModuleDescriptorParser.java:88)
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser$AbstractParser.getModuleDescriptor(AbstractModuleDescriptorParser.java:342)
 	at org.apache.ivy.plugins.parser.xml.XmlModuleDescriptorParser$Parser.getModuleDescriptor(XmlModuleDescriptorParser.java:180)
 	at org.apache.ivy.plugins.parser.xml.XmlModuleDescriptorParser.parseDescriptor(XmlModuleDescriptorParser.java:120)
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser.parseDescriptor(AbstractModuleDescriptorParser.java:48)
 	at org.apache.ivy.core.resolve.ResolveEngine.resolve(ResolveEngine.java:187)
 	at org.apache.ivy.Ivy.resolve(Ivy.java:508)
 	at org.apache.ivy.Main.run(Main.java:311)
 	at org.apache.ivy.Main.main(Main.java:219)
```

Switching to coursier might be an option to resolve this one?

Comment (alanbato) [reply build-support/ivy/ivy.xml]: @stuhood I'm not quite sure how to make this project use coursier, would you be willing to lend me a hand?

**Comment by grimreaper** [comment]
Thank you for your contribution. Unfortunately, we’re not going to continue maintaining twitter-commons and are archiving all pull requests.

**Pull request closed without merging**